src/Validators/NameValidator.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Portal\Validators;
+
+use Exception;
+
+class NameValidator
```

Comment:
can we rename this? it could be used to validate things that are not names, make it more generic as alpha validator perhaps? it could also go into the string validator class probably?